jvm-packages/xgboost4j-spark/src/test/scala/ml/dmlc/xgboost4j/scala/spark/XGBoostDFSuite.scala

```diff
@@ -18,14 +18,14 @@ package ml.dmlc.xgboost4j.scala.spark
 
 import ml.dmlc.xgboost4j.scala.{DMatrix, XGBoost => ScalaXGBoost}
 import ml.dmlc.xgboost4j.{LabeledPoint => XGBLabeledPoint}
-
 import org.apache.spark.ml.linalg.DenseVector
 import org.apache.spark.ml.param.ParamMap
-import org.apache.spark.rdd.RDD
 import org.apache.spark.sql._
-import org.scalatest.FunSuite
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.types.DataTypes
+import org.scalatest.{FunSuite, Matchers}
 
-class XGBoostDFSuite extends FunSuite with PerTest {
+class XGBoostDFSuite extends FunSuite with PerTest with Matchers {
```

Comment:
The rest of the suite does not use matchers. No saying you shouldn't, but maybe plain assert would look just as readable?

Reply:
Removed Matchers.